systori/apps/project/api.py

```diff
@@ -144,6 +208,11 @@ def week_by_day(self, request):
             ).data
         )
 
+    @swagger_auto_schema(
+        method="PUT",
+        request_body=SelectedDaySerializer,
+        responses={200: WrokerWithProjectsSerializer(many=True)},
```

Comment:
ok feels like this is the same as before, right? - is needed by apple tv app.

Reply:
if you're referring to the `@swagger_auto_schema` decorator then, it's only used for providing enough info to the `schema.yaml` generator. This doesn't affect the actual method being decorated.